detox/src/devices/runtime/factories/android.js

```diff
@@ -8,18 +8,21 @@ class RuntimeDriverFactoryAndroid extends RuntimeDeviceFactory {
     const apkValidator = serviceLocator.apkValidator;
     const fileXfer = serviceLocator.fileXfer;
     const devicePathBuilder = serviceLocator.devicePathBuilder;
+    const hashHelper = serviceLocator.hashHelper;
 
     const AppInstallHelper = require('../../common/drivers/android/tools/AppInstallHelper');
     const AppUninstallHelper = require('../../common/drivers/android/tools/AppUninstallHelper');
     const MonitoredInstrumentation = require('../../common/drivers/android/tools/MonitoredInstrumentation');
 
+
     return {
       ...commonDeps,
       adb,
       aapt,
       apkValidator,
       fileXfer,
       devicePathBuilder,
+      hashHelper,
```

Comment:
do you use it from here?
it's just a static method, so I dont think you need a service